i18n/en.json

```diff
@@ -98,7 +98,7 @@
     "library_scanning_enable_description": "Enable periodic library scanning",
     "library_settings": "External Library",
     "library_settings_description": "Manage external library settings",
-    "library_tasks_description": "Perform library tasks",
+    "library_tasks_description": "Scan external libraries for new and/or changed assets",
```

Comment:
We could expand this description a little more. Giving the details of all the actions of the job. Quite a few people have questions about what this job actually does.

Reply:
That's probably something that should go in the docs not the English translation of something.

Reply:
When seeing this job and the library's options on the library page, it is unclear when you should use which.

Reply:
> When seeing this job and the library's options on the library page, it is unclear when you should use which.

I agree and I first tried to have no button on the library job page since it wasn't very intuitive. Right now, though, the code requires it to compile.

Maybe I should just do the needed change to remove the button?

Longer term, I would like to move the various and often-confusing buttons from the jobs page to a more dedicated repair page. That page would show if there are assets missing thumbnails, missing metadata, ML etcetera, with a friendly button to fix. We could also stop showing assets without thumbs in the timeline altogether.

Reply:
> Maybe I should just do the needed change to remove the button?

Does this button ultimately run the scan command on `all` libraries? If yes, I guess we just need to clarify that in the description.

Reply:
Yes, it will run the scan on all libraries, and also run the check for pending library deletions

Reply:
I added a screenshot to the PR description